tests/cases/fourslash/importNameCodeFix_typeOnly2.ts

```diff
@@ -0,0 +1,19 @@
+/// <reference path="fourslash.ts" />
+
+// @importsNotUsedAsValues: error
+
+// @Filename: types.ts
+////export class A {}
+
+// @Filename: index.ts
+////const a: A = new A();
+
+goTo.file("index.ts");
+verify.codeFixAll({
+  fixAllDescription: ts.Diagnostics.Add_all_missing_imports.message,
+  fixId: "fixMissingImport",
+  newFileContent:
+`import { A } from "./types";
```

Comment:
why does this test file end with "_typeOnly2"?

Reply:
Since there is `@importsNotUsedAsValues: error`, when we see `const a: A`, we register the import fix as type-only. Then, we get to `new A()`, and we register an import fix as _not_ type-only. The point of this test is to ensure that when there is a conflict, we _don’t_ write the final import clause as type-only. It’s the natural next test case after `importNameCodeFix_typeOnly.ts` verifies the positive, though I see how the name might look confusing taken alone.